delfin/exception.py

```diff
@@ -332,3 +332,7 @@ class EmptyResourceMetrics(DelfinException):
 
 class TelemetryTaskExecError(DelfinException):
     msg_fmt = _("Failure in telemetry task execution")
+
+
+class ComponentNotFound(NotFound):
```

Comment:
Where is this exception used?

Reply:
It's used in EMC VNX driver, raised when naviseccli not found.

Reply:
currently I don't see this usage in VNX.. Are you planning to update VNX driver code in future PR ? hope it will be better to update both together

Reply:
yes, VNX driver code will be updated in the future PR.